Kitodo/src/test/java/org/kitodo/services/data/FilterServiceIT.java

```diff
@@ -169,290 +165,278 @@ public void shouldBuildQueryAndFindByProcessServiceByProcessTitle() throws Excep
         List<TaskDTO> taskDTOS = taskService.findByQuery(query, true);
-        assertEquals("Incorrect amount of open tasks with ordering 1!", 1, taskDTOS.size());
+        assertEquals("Incorrect amount of open tasks with ordering 1!", 2, taskDTOS.size());
 
-        query = filterService.queryBuilder("\"stepopen:Blocking\"", ObjectType.TASK, false, false, false);
+        query = filterService.queryBuilder("\"stepopen:Blocking\"", ObjectType.TASK, false, false);
         taskDTOS = taskService.findByQuery(query, true);
         assertEquals("Incorrect amount of open tasks with title 'Blocking'!", 1, taskDTOS.size());
 
-        query = filterService.queryBuilder("\"stepopen:Blocking\"", ObjectType.TASK, true, false, false);
+        query = filterService.queryBuilder("\"-stepopen:Blocking\"", ObjectType.TASK, false, false);
         taskDTOS = taskService.findByQuery(query, true);
-        assertEquals("Incorrect amount of open tasks with title 'Blocking'!", 0, taskDTOS.size());
-
-        query = filterService.queryBuilder("\"-stepopen:Blocking\"", ObjectType.TASK, true, false, false);
-        taskDTOS = taskService.findByQuery(query, true);
-        assertEquals("Incorrect amount of not open tasks with title different than 'Blocking'!", 2, taskDTOS.size());
+        assertEquals("Incorrect amount of not open tasks with title different than 'Blocking'!", 1, taskDTOS.size());
     }
 
     @Test
     public void shouldBuildQueryAndFindByTaskServiceByInProgressTasks() throws Exception {
         TaskService taskService = new ServiceManager().getTaskService();
 
-        QueryBuilder query = filterService.queryBuilder("\"stepinwork:3\"", ObjectType.TASK, false, false, false);
+        QueryBuilder query = filterService.queryBuilder("\"stepinwork:3\"", ObjectType.TASK, false, false);
         List<TaskDTO> taskDTOS = taskService.findByQuery(query, true);
         assertEquals("Incorrect amount of tasks in progress with ordering 3!", 1, taskDTOS.size());
 
-        query = filterService.queryBuilder("\"-stepinwork:3\"", ObjectType.TASK, false, false, false);
+        query = filterService.queryBuilder("\"-stepinwork:3\"", ObjectType.TASK, false, false);
         taskDTOS = taskService.findByQuery(query, true);
         assertEquals("Incorrect amount of tasks not in progress with ordering different than 3!", 2, taskDTOS.size());
 
-        query = filterService.queryBuilder("\"-stepinwork:2\"", ObjectType.TASK, false, false, false);
+        query = filterService.queryBuilder("\"-stepinwork:2\"", ObjectType.TASK, false, false);
         taskDTOS = taskService.findByQuery(query, true);
-        assertEquals("Incorrect amount of tasks not in progress with ordering different than 2!", 1, taskDTOS.size());
+        assertEquals("Incorrect amount of tasks not in progress with ordering different than 2!", 2, taskDTOS.size());
     }
 
+    @Ignore("problem with steplocked")
```

Comment:
What is the issue with this test?

Reply:
Not sure if this filter should be removed or changes in the code should allow this type of search.

Reply:
In 2.x branch you can see on "Meine Aufgaben" page all open and locked by yourself and other users tasks regarding to your assigned projects and rights to this task through user / user group assignments of this task.

If this is changed that locked tasks by other users should not visible for current user then the code and / or test must be adjusted.

Reply:
Probably the best is add ticket for it, which defines how exactly should it work. It is not directly related to split processes and templates.

Reply:
Ticket is added.